pkgs/by-name/up/uplosi/package.nix

```diff
@@ -0,0 +1,34 @@
+{ lib
+, fetchFromGitHub
+, buildGo121Module
+}:
+let
+  version = "0.1.1";
+  pname = "uplosi";
+in
+buildGo121Module {
+  inherit pname version;
```

Comment:
```suggestion

buildGo121Module rec {
  version = "0.1.1";
  pname = "uplosi";
```

Reply:
Done. Why are we using rec if we don't have to? https://nix.dev/recipes/best-practices#recursive-attribute-set-rec

Reply:
I'd argue that `overrideAttrs` is unlikely to work in both cases. However, if one day `buildGoModule (finalAttrs: { })` is implemented, it'll be simpler to rewrite from `buildGoModule rec { }`.